### PR TITLE
#18474 `Reset UI settings` improvements

### LINKED ISSUE #18474: Rethink the `Reset UI settings` and `Reset workspace state` actions
**Is your feature request related to a problem? Please describe.**
Under the hood, these two actions do the same. They confuse users and don't do their only purpose.

**Describe the solution you'd like**
We must rethink how these actions are presented in UI and how they behave.

***Note: the following changes must be reflected on the wiki pages ([1](https://github.com/dbeaver/dbeaver/wiki/Reset-UI-settings), [2](https://github.com/dbeaver/dbeaver/wiki/Reset-workspace)), possibly linked together.***

We want the following:
1. Remove the `Reset workspace state` action.
1. Move the `Reset UI settings` under the `Help` menu.
3. Add an additional action called `Clear history`.

By clicking on the `Reset UI settings`, a confirmation dialog will be shown describing what is about to happen, including listings of what will actually be reset among UI settings (perspective, theme, fonts, etc.) and other preferences (depending on the installed plugins).

By clicking on the `Clear history`, a dialog with the following checkboxes will be shown:
- Clear query history - will truncate the QMDB database. It can be done from the preferences, but it's hard to find.
- Clear logs - will remove all logs, both normal and debug. No one needs them until asked for.
- Clear task run statistics - task execution logs you probably really need.

After choosing the appropriate action, the user will press the `Apply and Restart` button to perform the required actions and then restart DBeaver.

### PR DESCRIPTION
**Pull request by ShadelessFox**
Actions were moved to the `Help` menu:

java_UZ6IIB6VpO

|Reset UI settings|Clear History|
|---|---|
|java_V8fHTc7hMa|java_M09LxwMWZB|

The planned option to clear log files wasn't implemented because it depends on #18205.

Acceptance criteria:
- [x] The `.plugins` folder is deleted completely from the `.metadata` folder of the DBeaver workspace after resetting UI settings
- [ ] The query manager is empty after clearing it
- [x] Task run statistics is empty after clearing it

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.ui.app.standalone/plugin.xml [hunk 1]
```diff
@@ -93,6 +93,16 @@
...
+        <command
+            id="org.jkiss.dbeaver.core.util.resetSettings"
+            categoryId="org.jkiss.dbeaver.core.util"
+            name="Reset UI Settings..."
+            description="Reset UI settings and restart"/>
+        <command
+            id="org.jkiss.dbeaver.core.util.clearHistory"
+            categoryId="org.jkiss.dbeaver.core.util"
+            name="Clear History..."
+            description="Clear logs, query history and restart"/>
```
Comment (E1izabeth) [inline plugins/org.jkiss.dbeaver.ui.app.standalone/plugin.xml:105]: Please, add localization.

**Comment by uslss** [comment]
Query log history option isn't available
image

**Comment by ShadelessFox** [comment]
It's available only in PRO products because the query history is only saved there.

**Comment by uslss** [comment]
verified

**Pull request merged**